README.md

```diff
@@ -2,10 +2,35 @@
 
 This repository contains example notebooks associated with datasets, conference tools, or more generally notebooks that illustrate the use of data on DANDI. This repository is cloned into the [DANDI JupyterHub environment](https://hub.dandiarchive.org). Please note that you will need to visit https://dandiarchive.org and sign in once to get access to the JupyterHub.
 
-To add a new notebook, please send a PR. For specific dandiset related analysis, place your notebooks or other code into:
+## Submission instructions
+To add new notebooks, please send a Pull Request. Submissions should use the following file structure:
 
-`<dandiset id>/<org or lab name>/<mnemonic for paper or analysis>/<code>`
+```
+example-notebooks/
+└── <dandiset id>/
+    └── <org or lab name>/
+        └── <mnemonic for paper or analysis>/
+            ├── environment.yml
+            ├── README.md
+            ├── <analysis 1>.ipynb
+            ├── <analysis 2>.ipynb
+            ├── ...
+            └── <analysis n>.ipynb
+```
 
 For example, [000055/bruntonlab/peterson21](https://github.com/dandi/example-notebooks/tree/9b1fb88667595a3abcdefda46bbe08e538dcbf0f/000055/BruntonLab/peterson21)
```

Comment:
This I think might work
```suggestion
For example, [000055/bruntonlab/peterson21](./000055/BruntonLab/peterson21)
```
and thus point people to current, not potentially outdated version which might gain some fixes since now